backend/YouTubeV2.Application/Services/JwtFeatures/JwtHandler.cs

```diff
@@ -36,7 +35,7 @@ private async Task<ICollection<Claim>> GetClaimsAsync(User user)
         {
             var claims = new List<Claim>
             {
-                new Claim(ClaimTypes.Name, user.Email),
+                new Claim(ClaimTypes.Name, user.Email!),
```

Comment:
will we ever use email?

Reply:
i dont think so, roles + id should be enough